server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileLoader.java

```diff
@@ -91,48 +88,40 @@ public QualityProfileDto getByLangAndName(String lang, String name) {
 
   @CheckForNull
   public ActiveRule getActiveRule(ActiveRuleKey key) {
-    return index.get(ActiveRuleIndex.class).getNullableByKey(key);
+    return activeRuleIndex.getNullableByKey(key);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "getNullableByKey"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)